pandas/tests/io/formats/style/test_tooltip.py

```diff
@@ -83,3 +87,98 @@ def test_tooltip_css_class(styler):
         props="color:green;color:red;",
     ).to_html()
     assert "#T_ .another-class {\n  color: green;\n  color: red;\n}" in result
+
+
+@pytest.mark.parametrize(
+    "ttips",
```

Comment:
Can you parametrize over `"data, columns, index"` and in the test body define `ttips = DataFrame(data=data, columns=columns, index=index)`?